README.md

```diff
@@ -30,8 +30,9 @@ Links
 * [Submit a patch](CONTRIBUTING.md)
 
 *Note*: For questions related to ITK, please use the official [Discussion](https://discourse.itk.org/)
-space: GitHub issues are reserved to track different aspects of the software
-development process, as highlighted by the available templates.
+space: the [issue tracking](https://github.com/InsightSoftwareConsortium/ITK/issues)
```

Comment:
issue tracking -> issue tracker

is correct grammar.

Reply:
Done in [66e9336](https://github.com/InsightSoftwareConsortium/ITK/commit/66e933613cc4390755bf0303fb04ee61771173e1).

I had thought about it, but opted for using the exact same term as in https://github.com/InsightSoftwareConsortium/ITK/blob/27f450358cd18c442ed57cbef918b7feafb15823/README.md?plain=1#L29. But I understand the it makes sense in the list, but maybe not here due to the grammar reasoning. They both point to the same URL, so it should be clear for users that they refer to the same thing.

Thanks Matt.